sapi/fpm/fpm/fpm_status.c

```diff
@@ -45,6 +46,97 @@ int fpm_status_init_child(struct fpm_worker_pool_s *wp) /* {{{ */
 }
 /* }}} */
 
+/* {{{ proto array fastcgi_get_status
+ * Returns the status of the fastcgi process manager */
+PHP_FUNCTION(fastcgi_get_status)
```

Comment:
I would prefer to have a function that would just fill the supplied `zval` and `PHP_FUNCTION` in `fpm_main.c` that would call it. It means function like:
```
int fpm_status_export_to_zval(zval *status);
```

Reply:
Changed in 979cec172be7eb0f3553617e1ce58d12adf3b1c0